ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -168,8 +158,19 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                         resource.hook = status.hook;
                                         resource.requiresPruning = status.requiresPruning;
                                     }
-                                    resources.push(resource);
+                                    resources.set(node.uid, resource);
                                 });
+                                const resourcesRef = Array.from(resources.values());
+                                return resourcesRef;
+                            };
+
```

Comment:
You want to add back the orphaned nodes here, so they will be counted?

Reply:
only if `pref.orphanedResources` is true i.e. if the check box for showing orphaned is ticked it will get counted, yes.